packages/devtools_app_shared/lib/src/ui/theme/theme.dart

```diff
@@ -71,61 +73,63 @@ ThemeData _baseTheme({
-        padding: const EdgeInsets.all(densePadding),
-        minimumSize: Size(buttonMinWidth, defaultButtonHeight),
-        fixedSize: Size.fromHeight(defaultButtonHeight),
+      outlinedButtonTheme: OutlinedButtonThemeData(
+        style: OutlinedButton.styleFrom(
+          minimumSize: Size(buttonMinWidth, defaultButtonHeight),
+          fixedSize: Size.fromHeight(defaultButtonHeight),
+          foregroundColor: theme.colorScheme.onSurface,
+          padding: const EdgeInsets.symmetric(horizontal: denseSpacing),
+        ),
       ),
-    ),
-    elevatedButtonTheme: ElevatedButtonThemeData(
-      style: ElevatedButton.styleFrom(
-        minimumSize: Size(buttonMinWidth, defaultButtonHeight),
-        fixedSize: Size.fromHeight(defaultButtonHeight),
-        backgroundColor: theme.colorScheme.primary,
-        foregroundColor: theme.colorScheme.onPrimary,
-        padding: const EdgeInsets.symmetric(horizontal: denseSpacing),
+      textButtonTheme: TextButtonThemeData(
+        style: TextButton.styleFrom(
+          padding: const EdgeInsets.all(densePadding),
+          minimumSize: Size(buttonMinWidth, defaultButtonHeight),
+          fixedSize: Size.fromHeight(defaultButtonHeight),
+        ),
       ),
-    ),
-    menuButtonTheme: MenuButtonThemeData(
-      style: ButtonStyle(
-        textStyle: WidgetStatePropertyAll<TextStyle>(theme.regularTextStyle),
-        fixedSize: const WidgetStatePropertyAll<Size>(Size.fromHeight(24.0)),
+      elevatedButtonTheme: ElevatedButtonThemeData(
+        style: ElevatedButton.styleFrom(
+          minimumSize: Size(buttonMinWidth, defaultButtonHeight),
+          fixedSize: Size.fromHeight(defaultButtonHeight),
+          backgroundColor: theme.colorScheme.primary,
+          foregroundColor: theme.colorScheme.onPrimary,
+          padding: const EdgeInsets.symmetric(horizontal: denseSpacing),
+        ),
       ),
-    ),
-    dropdownMenuTheme: DropdownMenuThemeData(
-      textStyle: theme.regularTextStyle,
-    ),
-    progressIndicatorTheme: ProgressIndicatorThemeData(
-      linearMinHeight: defaultLinearProgressIndicatorHeight,
-    ),
-    primaryTextTheme: _devToolsTextTheme(theme, theme.primaryTextTheme),
-    textTheme: _devToolsTextTheme(theme, theme.textTheme),
-  );
+      menuButtonTheme: MenuButtonThemeData(
+        style: ButtonStyle(
+          textStyle: WidgetStatePropertyAll<TextStyle>(theme.regularTextStyle),
+          fixedSize: const WidgetStatePropertyAll<Size>(Size.fromHeight(24.0)),
+        ),
+      ),
+      dropdownMenuTheme: DropdownMenuThemeData(
+        textStyle: theme.regularTextStyle,
+      ),
+      progressIndicatorTheme: ProgressIndicatorThemeData(
+        linearMinHeight: defaultLinearProgressIndicatorHeight,
+      ),
+      primaryTextTheme: _devToolsTextTheme(theme, theme.primaryTextTheme),
+      textTheme: _devToolsTextTheme(theme, theme.textTheme),
+      colorScheme: theme.colorScheme.copyWith(
+        surface: backgroundColor,
```

Comment:
do we really want to override the surface color specified by the `darkColorScheme` and `lightColorScheme` definitions?

Reply:
For the darkColorScheme and lightColorScheme, the surface color is the same as this backgroundColor. This makes sure that the surface color for IDE themes matches the IDE's background color.